src/components/AccountForm.vue

```diff
@@ -392,7 +392,8 @@ export default {
 				return this.loadingMessage ?? t('mail', 'Connecting')
 			}
 			if (this.mode === 'manual' && this.useOauth) {
-				return this.account ? t('mail', 'Reconnect Google account') : t('mail', 'Sign in with Google')
+				if(this.isGoogleAccount)return this.account ? t('mail', 'Reconnect Google account') : t('mail', 'Sign in with Google')
+				else return this.account ? t('mail', 'Reconnect Microsoft account') : t('mail', 'Sign in with Microsoft')
```

Comment:
use a block for the if body please
```suggestion
				if (this.isGoogleAccount) {
				    return this.account ? t('mail', 'Reconnect Google account') : t('mail', 'Sign in with Google')
				} else {
				    return this.account ? t('mail', 'Reconnect Microsoft account') : t('mail', 'Sign in with Microsoft')
				}
```